plugins/nginx_ingress.yaml

```diff
@@ -0,0 +1,129 @@
+version: 0.0.1
+title: Nginx Ingress
+description: Log parser for Nginx Ingress for Kubernetes
+supported_platforms: 
```

Comment:
Should this also work with openshift ingress?

Reply:
We will add openshift support in future.